app/src/renderer/js/editor.js

```diff
@@ -92,7 +94,7 @@ document.addEventListener('DOMContentLoaded', () => {
     if (preview.duration) {
       const timeAtEvent = getTimestampAtEvent(event, preview.duration);
       previewTimeTip.style.left = `${event.pageX}px`;
-      previewTimeTip.textContent = `${moment().startOf('day').milliseconds(timeAtEvent * 1000).format('m:ss.SS')}`;
+      previewTimeTip.textContent = `${moment().startOf('day').milliseconds(timeAtEvent * 1000).format('m:ss.SS')} (${moment().startOf('day').milliseconds(currentPreviewDuration * 1000).format('m:ss.SS')})`;
```

Comment:
I think we should have a common function to format these, I also think it's quicker to reuse the same moment objects rather than creating new ones, but this could be for another PR if we want to get this in quick 👍

Reply:
Agreed, but for the beta, this should be fine 😄